app/views/layouts/application.html.erb

```diff
@@ -17,7 +17,12 @@
     <section class="top-bar-section">
       <ul class="right">
         <%- if current_user -%>
-          <li><%= link_to 'Sign Out', destroy_user_session_path, method: :delete %></li>
+          <%= link_to "#{current_user.username}", "#",
```

Comment:
This is pretty minor, but there's no need to use string interpolation here! You can just do `link_to current_user.username, ...`